Sources/Commands/SwiftTool.swift

```diff
@@ -559,6 +562,7 @@ public class SwiftTool {
         let delegate = ToolWorkspaceDelegate(self.stdoutStream, isVerbose: isVerbose, diagnostics: diagnostics)
         let provider = GitRepositoryProvider(processSet: processSet)
         let cachePath = self.options.useRepositoriesCache ? try self.getCachePath() : .none
+        _  = try self.getConfigPath() // TODO: actually use this in the workspace 
```

Comment:
a follow up PR will make use of this in Workspace, which I hope would trigger the right calls in any client (like IDEs) cc @abertelrud

Reply:
Thanks for the heads-up.  Yes, clients would probably get this for free if they call `Workspace.Configuration()` to get the value to pass down in the `config:` parameter when instantiating the workspace.

Reply:
Actually, that config directory is the package-local `.swiftpm` directory, I think.  So this might require clients to change.  I'll take a closer look at this.

Reply:
right, this PR does not impact the package-local config directory... this will come in subsequent PRs